domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/service/behandling/BehandlingServiceImpl.kt

```diff
@@ -180,7 +184,12 @@ class BehandlingServiceImpl(
             logger.warn { "Navident ${saksbehandler.navIdent} med rollene ${saksbehandler.roller} har ikke tilgang til å sende behandling til beslutter" }
             return KanIkkeSendeTilBeslutter.MåVæreSaksbehandler.left()
         }
+        val oppgaveId = søknadRepo.hentOppgaveIdForSoknad(behandlingId)
         return hentBehandling(behandlingId, saksbehandler, correlationId).tilBeslutning(saksbehandler).also {
+            oppgaveId?.let { id ->
+                logger.info { "Ferdigstiller oppgave med id $id for søknad med behandlingsId $behandlingId" }
```

Comment:
Jeg og Ramzi startet også på denne måten i supplerende stønad uføre. Men det saksbehandlerne egentlig ville ha, var den samme oppgaven fra søknaden kom inn til behandlingen var avsluttet (iverksatt/avbrutt), så vi endret alle flytene våre til sistnevnte og fikk mye skryt for det.

Reply:
Også endret vi etterhvert lukking av oppgaver som "best effort" og logget dersom vi ikke fikk det til, så lukking av oppgaver ikke skulle stoppe opp behandlingen av en sak.

For noen type behandlinger laget vi også jobber som lukket oppgavene.

Reply:
Taulant ville ha det sånn som dette, så jeg tenkte han kunne få se det sånn først, og så kan jeg evt endre det etterpå hvis det ikke var sånn han ville ha det likevel når han får se det 😅 Vi kan enten beholde oppgaven til det er iverksatt, eller endre på oppgaven sånn som du snakket om slik at den får en annen type for når saken går til beslutter. For meg er det også mer naturlig å avslutte oppgaven når saken faktisk er ferdig.